harness/tests/experiment/fixtures/pytorch_xor_model.py

```diff
@@ -405,3 +405,4 @@ def train_batch(
             self.context.step_optimizer(self.optimizer)
 
         return {"loss": loss}
+
```

Comment:
extraneous newline

Reply:
Oops! I'll run `make check` and/or the like, don't worry.